install_config/persistent_storage/persistent_storage_flex_volume.adoc

```diff
@@ -127,60 +125,62 @@ It contains:
 
-Operations should be idempotent. The attachment of an already attached volume
-or the mounting of an already mounted volume should result in a successful
-operation.
+Operations should be idempotent, which means, that the attachment of an already
+attached volume or the mounting of an already mounted volume should result in a
+successful operation.
+
+The FlexVolume driver can work in two modes:
+
+* xref:flex-volume-drivers-with-master-initiated-attach-detach[with the master-initated attach/detach] operation, or
+* xref:flex-volume-drivers-without-master-initiated-attach-detach[without the master-initated attach/detach] operation.
 
-The Flex Volume driver can work in two modes, with the attach/detach operation or
-without it. The attach/detach operation is used by {product-title} master
-to attach a volume to a node and to detach it from a node. This
-is useful when a node becomes unresponsive from any reason. Then, the master can
-kill all pods on the node, detach all volumes from it, and attach the volumes to
-other nodes to resume the applications while the original node is still not
-reachable.
+The `attach/detach` operation is used by the {product-title} master to attach a
+volume to a node and to detach it from a node. This is useful when a node
+becomes unresponsive from any reason. Then, the master can kill all pods on the
+node, detach all volumes from it, and attach the volumes to other nodes to
+resume the applications while the original node is still not reachable.
 
 [IMPORTANT]
 ====
-Not every storage back-end supports master-initiated detachment of a volume from
-another machine. Carefully consider what kind of Flex Volume driver you are
-going to write.
+Not all storage back-end supports master-initiated detachment of a volume from another machine.
 ====
 
 [[flex-volume-drivers-with-master-initiated-attach-detach]]
-==== Flex Volume drivers with master-initiated attach/detach
+=== FlexVolume Drivers with Master-initiated Attach/Detach
 
-A Flex Volume driver that supports master-controlled attach/detach must implement
-these operations:
+A FlexVolume driver that supports master-controlled attach/detach must implement
+following operations:
 
 `init`::
-* Initializes the driver. It is called during initialization of masters and nodes.
+Initializes the driver. It is called during initialization of masters and nodes.
+
 * Arguments: none
 * Executed on: master, node
 * Expected output: default JSON
 
+
 `getvolumename`::
-* Returns the unique name of the volume. This name must be consistent among all
-masters and nodes and will be used in subsequent `detach` call as
-`<volume-name>`. Any `'/'` character in the `<volume-name>` are automatically
-replaced by `'~'`.
+Returns the unique name of the volume. This name must be consistent among all
+masters and nodes and because it is used in subsequent `detach` call as
+`<volume-name>`. Any `/` characters in the `<volume-name>` are automatically
```

Comment:
I'm not sure if something is missing from the "because it is used in subsequent detach call..." sentence. Maybe:

"masters and nodes and because it is used in subsequent `detach` _calls_ as <volume-name>."
or
"masters and nodes, because it is used in subsequent `detach` call as <volume-name>."